services/dydx_client/withdrawal.py

```diff
@@ -27,11 +31,37 @@ def slow_withdrawal(self, withdrawal_params):
         return vars(withdrawal)
 
     def fast_withdrawal(self, withdrawal_params):
-        fast_withdrawal = self.client.private.create_fast_withdrawal(
-            **withdrawal_params
+        withdrawal_amount = withdrawal_params["withdrawal_amount"]
+        fast_withdrawal_result = self.get_transfer(withdrawal_amount)
+        lp_position_id_result = list(
+            fast_withdrawal_result.data["liquidityProviders"].keys()
+        )[0]
+        quote = fast_withdrawal_result.data["liquidityProviders"][
+            lp_position_id_result
+        ]["quote"]
+        if quote is None:
+            raise Exception("Could not get a quote")
+        debit_amount = quote["debitAmount"]
+        create_fast_withdrawal_result = self.client.private.create_fast_withdrawal(
+            position_id=withdrawal_params["position_id"],
+            credit_asset=constants.ASSET_USDC,
+            credit_amount=withdrawal_amount,
+            debit_amount=debit_amount,
+            to_address=withdrawal_params["to_address"],
+            lp_position_id=lp_position_id_result,
+            lp_stark_public_key=list(
+                fast_withdrawal_result.data["liquidityProviders"].values()
+            )[0]["starkKey"],
+            expiration=epoch_seconds_to_iso(time.time() + 604801),
         )
-        return vars(fast_withdrawal)
+        return create_fast_withdrawal_result.data
 
     def transfer(self, params):
         transfers = self.client.private.get_transfers(**params)
         return vars(transfers)
```

Comment:
Can you paste an example of params here ?

Reply:
this params will contained  req body for get_transfer that is limit and transfer_type

Reply:
> this params will contained req body for get_transfer that is limit and transfer_type

`{
limit:100,
Transfer_type:withdrawals /deposit 
}`